homestar-runtime/src/metrics/exporter.rs

```diff
@@ -0,0 +1,31 @@
+//! Metrics Prometheus recorder.
+
+use crate::{metrics::node, settings};
+use metrics_exporter_prometheus::{Matcher, PrometheusBuilder};
+use std::net::{IpAddr, Ipv4Addr, SocketAddr};
+
+/// Set up Prometheus buckets for matched metrics and install recorder.
+pub(crate) fn setup_metrics_recorder(settings: &settings::Monitoring) -> anyhow::Result<()> {
```

Comment:
minor: same, comment.

Reply:
This one is documented. Something that needs to change with it?